stl/inc/xcharconv_ryu.h

```diff
@@ -1175,7 +1184,8 @@ _NODISCARD inline __floating_decimal_32 __f2d(const uint32_t __ieeeMantissa, con
   return __fd;
 }
 
-_NODISCARD inline to_chars_result _Large_integer_to_chars(char* const _First, char* const _Last,
+template <class _CharT>
+_NODISCARD inline pair<_CharT*, errc> _Large_integer_to_chars(_CharT* const _First, _CharT* const _Last,

```

Comment:
`inline` is unnecessary after `template`:
```suggestion
template <class _CharT>
_NODISCARD pair<_CharT*, errc> _Large_integer_to_chars(_CharT* const _First, _CharT* const _Last,
```